package.json

```diff
@@ -17,7 +17,7 @@
     "clean": "npx turbo run clean && rimraf .turbo && rimraf *.tsbuildinfo",
     "clean:full": "npx turbo run clean:full && rimraf .turbo && rimraf *.tsbuildinfo && rimraf node_modules",
     "dev": "turbo run dev",
-    "deploy": "yarn install && yarn clean && yarn workspace cdk bootstrap && yarn workspace cdk deploy:no-review",
+    "deploy": "yarn install && yarn clean && ./deployCdk.sh",
```

Comment:
Does it work cross platform? e.g. Windows?
What you think about having the script call a node script like `"deploy": ""yarn install && yarn clean && node ./script-to-call.js"`
in the `script-to-call.js`, it can also use packages like `https://www.npmjs.com/package/inquirer` to enhance the experience (not sure if necessary, just saying it would give us the flexibility to do it)

Reply:
I'll have to test on Windows. I did originally write a node script with inquirer, but I couldn't figure out how to pass the authMode selection from the node script to cdk as a context argument. I tried using environment variables but it seemed like the node script ran on a different process and I couldn't then read the environment variable in the cdk context. Let me know if you have any ideas and I can add it back I still have the node script.

Reply:
Got the node script working and confirmed the node script for auth selection works on Windows. I did find Windows failing to deploy but it did not have to do with this change, so will follow up with that fix too.